src/shared/reducers/reducer.js

```diff
@@ -60,16 +62,17 @@ const initialState = {
   },
 
   events: toByIdMap(
-    norcalMtb2016Events
-    .concat(ncnca2016Events)
-    .concat(ncnca2017Events)
-    .concat(usac2017Events)
+    norcalMtb2016Events.map(setEventsCalendarId('cal-norcal-mtb-2016'))
+    .concat(ncnca2016Events.map(setEventsCalendarId('cal-ncnca-2016')))
+    .concat(ncnca2017Events.map(setEventsCalendarId('cal-ncnca-2017')))
+    .concat(usac2017Events.map(setEventsCalendarId('cal-usac-2017')))
   ),
 
   //calenars map by id
   calendars: {
     ['cal-norcal-mtb-2016']: {
       id: 'cal-norcal-mtb-2016',
+      slug: 'norcal-mtb',
```

Comment:
Slug is actually a calculated property, you can see how it's done here https://github.com/Restuta/rcn.io/blob/master/src/client/calendar/utils/get-calendar-id.js in reverse.

Convention is `slug = calendarId.replace('cal-', '')`. So if you go on live

https://rcn.io/calendars/usac-2017 (is canonical url)
https://rcn.io/calendars/cal-usac-2017 (also available, but we don't link to it anywhere and ideally should redirect to non-cal version)

So part of calendar id after `cal-` serves as slug, therefore you don't ave to store it. 

`norcal-mtb` breaks this rule and I would suggest renaming it and setting redirect in router from old route (`norcal-mtb`) to `norcal-mtb-2016`.

Reply:
@Restuta maybe create a helper function getCalendarsSlug(calendar) and use it everywhere

Reply:
Yeah that makes sense.

Reply:
Done